pytentiostat/config_reader.py

```diff
@@ -64,13 +100,53 @@ def get_lsv_params(config_data):
 
 
 def get_ca_params(config_data):
+    """
+    Obtains the parameters for a chronoamperometry experiment
```

Comment:
I'm not so sure about these guys, but I think ``Returns the parameters for a chronoamperometry experiment from the config data``

i.e., returns is maybe better than obtains?  But I am not so clear on that.

Reply:
I think that phrasing is more clear. I changed it on the other functions that do similar things as well so they would be consistent.